zookeeper-api/src/main/java/org/apache/helix/zookeeper/impl/client/FederatedZkClient.java

```diff
@@ -79,7 +83,31 @@
   private volatile boolean _isClosed;
   private PathBasedZkSerializer _pathBasedZkSerializer;
 
+  public FederatedZkClient() {
+    this(new RealmAwareZkClient.RealmAwareZkClientConfig()
+        .setZkSerializer(new ZNRecordSerializer()));
+  }
+
   // TODO: support capacity of ZkClient number in one FederatedZkClient and do garbage collection.
+  public FederatedZkClient(RealmAwareZkClient.RealmAwareZkClientConfig clientConfig) {
+    try {
```

Comment:
FederatedZkClient should take in connectionConfig and clientConfig. I'll just assume this is a placeholder.